### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by pmwtsn**

### CODE REVIEW

File: guide/example.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+
+## Import the dictionary
+
+Import the dictionary directly from S3:
+
+    $ ivory import
+
+Display the dictionary:
+
+    $ ivory ???
```
Comment (charleso) [inline guide/example.md]: `ivory cat-dictionary`

File: guide/example.md [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+This worked example demonstrates a simple workflow for using Ivory.
+
+Once you have installed Ivory,
+
+
+## Create an Ivory repo
+
+Create a new Ivory repository:
+
+    $ ivory ???
```
Comment (charleso) [inline guide/example.md]: `ivory create-repository -z your/timezone /path/to/repo`

https://github.com/ambiata/ivory/blob/master/ivory-performance/src/test/bin/run#L59

File: guide/example.md [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+
+
+
+## Import the dictionary
+
+Import the dictionary directly from S3:
+
+    $ ivory import
```
Comment (charleso) [inline guide/example.md]: https://github.com/ambiata/ivory/blob/master/ivory-performance/src/test/bin/run#L62

**Comment by charleso** [comment]
:+1:

**Comment by nhibberd** [comment]
@charles bump. Can you rebase and merge

**Pull request merged**